gtsfm/multi_view_optimizer.py

```diff
@@ -46,14 +45,14 @@ def create_computation_graph(
         self,
         images_graph: List[Delayed],
         num_images: int,
-        keypoints_graph: List[Delayed],
+        keypoints_graph: List[Keypoints],
         i2Ri1_graph: Dict[Tuple[int, int], Delayed],
         i2Ui1_graph: Dict[Tuple[int, int], Delayed],
         v_corr_idxs_graph: Dict[Tuple[int, int], Delayed],
         all_intrinsics: List[Optional[gtsfm_types.CALIBRATION_TYPE]],
         absolute_pose_priors: List[Optional[PosePrior]],
         relative_pose_priors: Dict[Tuple[int, int], PosePrior],
-        two_view_reports_dict: Optional[Dict[Tuple[int, int], TwoViewEstimationReport]],
+        two_view_reports_dict: Optional[Dict[Tuple[int, int], Delayed]],
```

Comment:
Confirming that the type is indeed Delayed, per

https://github.com/borglab/gtsfm/blob/master/gtsfm/scene_optimizer.py#L254

These come out of the TwoViewEstimator as Delayed objects:
https://github.com/borglab/gtsfm/blob/master/gtsfm/two_view_estimator.py#L382